pmm-app/tests/verifyPMMInventory_test.js

```diff
@@ -0,0 +1,179 @@
+Feature('Inventory page');
+
+Before(async (I) => {
+  I.Authorize();
+});
+
+Scenario(
+  'PMM-T371 - Verify sorting in Inventory page(Services tab) @not-pr-pipeline @not-ui-pipeline @nightly',
+  async (I, pmmInventoryPage) => {
+    I.amOnPage(pmmInventoryPage.url);
+    I.waitForVisible(pmmInventoryPage.fields.tableRow, 20);
+    const countOfRows = await I.grabNumberOfVisibleElements(pmmInventoryPage.fields.tableRow);
+    let serviceNames = new Array(countOfRows);
+    let forSort = new Array(countOfRows);
+
+    for (i = 1; i <= serviceNames.length; i++) {
+      serviceNames[i - 1] = await pmmInventoryPage.getCellValue(i, 4);
+      forSort[i - 1] = serviceNames[i - 1];
+    }
+    forSort.sort();
```

Comment:
This test logic seems to be far more complicated than needed. We could just as well grab all Inventory Page elements and see if they are sorted. Don't understand why you should call `forSort.sort()`.